examples/route-mirroring/docker-compose.yaml

```diff
@@ -0,0 +1,57 @@
+version: "3.8"
+services:
+
+  # base images
+  flask_service:
+    build:
+      context: ../shared/flask
+    restart: "no"
+    deploy:
+      replicas: 0
+
+  # front-proxy
+  front-envoy:
+    build:
+      context: .
+      dockerfile: Dockerfile-frontenvoy
+    ports:
+    - "${PORT_PROXY:-8080}:8080"
+    - "${PORT_HTTPS:-8443}:8443"
+    - "${PORT_STATS:-8001}:8001"
+    depends_on:
+      - service1
+      - service1-mirror
+      - service2
+      - service2-mirror
+
+  service1:
+    build:
+      context: ../shared/tracing
```

Comment:
please dont use `shared/tracing` im in the process of removing it altogether - not least because it has multiple procs in the container

rather separate out any backend service to its own container (see eg the zipkin sandbox for an example of that)